PolyEngine/Core/Src/pe/core/storage/PriorityQueue.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <pe/Defines.hpp>
+
+namespace pe::core::storage
+{
+	template<class _Ty,class _Pr = std::less<typename std::vector<_Ty>::value_type> >
```

Comment:
I would stick to normal engine conventions for template parameters, underscores are better left off in the stl implementation